File: app/src/main/java/com/boardgamegeek/ui/DataActivity.java
package com.boardgamegeek.ui;

import android.support.v4.app.Fragment;

import com.boardgamegeek.R;

public class DataActivity extends TopLevelSinglePaneActivity {

	@Override
	protected Fragment onCreatePane() {
		return new DataFragment();
	}

	@Override
	protected int getDrawerResId() {
		return R.string.title_backup;
	}
}

File: app/src/main/java/com/boardgamegeek/ui/DataActivity.kt
package com.boardgamegeek.ui

import androidx.fragment.app.Fragment

import com.boardgamegeek.R
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class DataActivity : TopLevelSinglePaneActivity() {
    override val firebaseContentType = "Data"

    override fun onCreatePane(): Fragment = DataFragment()

    override val navigationItemId: Int = R.id.data
}
